examples/train.py

```diff
@@ -53,25 +55,26 @@ def train_step(self, data, optimizer):
     ])
     trainset = CIFAR10(
         root='data', train=True, download=True, transform=transform)
+
     trainloader = DataLoader(
         trainset, batch_size=128, shuffle=True, num_workers=2)
 
-    optimizer = optim.SGD(model.parameters(), lr=0.001, momentum=0.9)
```

Comment:
Why should we modify these parameters? Such as type of optimize, max_epochs, interval, .etc.

Reply:
Using adam can get faster convergence in this case. That is an optional modification.